storm-core/src/jvm/org/apache/storm/command/AdminCommands.java

```diff
@@ -109,6 +133,7 @@ public void printCliHelp(String command, PrintStream out) {
     static {
         COMMANDS.put("remove_corrupt_topologies", new RemoveCorruptTopologies());
         COMMANDS.put("zk_cli", new ZkCli());
+        COMMANDS.put("creds", new CredentialsDebug());
```

Comment:
is there documentation that should be updated for these commands?

Reply:
+1 If we don't have any documentation for admin commands, it should be useful to have one, otherwise end users wouldn't even know its existence.